docs/content/users/install/shell-completion.md

```diff
@@ -2,22 +2,65 @@
 
 Most people like to have shell completion on the command line. In other words, when you're typing a command, you can hit `<TAB>` and the shell will show you what the options are. For example, if you type `ddev <TAB>`, you'll see all the possible commands. `ddev debug <TAB>` will show you the options for the command. And `ddev list -<TAB>` will show you all the flags available for [`ddev list`](../usage/commands.md#list).
 
-Shells like Bash and Zsh need help to do this though, they have to know what the options are. DDEV provides the necessary hint scripts, and if you use Homebrew, they get installed automatically.
+Shells like Bash and Zsh need help to do this though, they have to know what the options are, and DDEV provides the necessary hint scripts.
 
 ## macOS Bash with Homebrew
 
-The easiest way to use Bash completion on macOS is install it with Homebrew ([docs](https://docs.brew.sh/Shell-Completion#configuring-completions-in-bash)). `brew install bash-completion`. When you install it though, it will warn you with something like this, which **may vary on your system**. Add the following line to your `~/.bash_profile` file (or if that doesn't exist, to your `~/.profile`:
+To make Homebrew completions available in Bash the Homebrew-managed path, either `etc/profile.d` or `etc/bash_completion.de` has to be sourced.
 
-```
-[[ -r "$(brew --prefix)/etc/profile.d/bash_completion.sh" ]] && . "$(brew --prefix)/etc/profile.d/bash_completion.sh"
+=== "macOS Bash"
+
+     Add the following block to your `~/.bashprofile` or if it doesn't exist to your `~/.profile` file (see [docs](https://docs.brew.sh/Shell-Completion#configuring-completions-in-bash)):
+
+    ```bash
+    if type brew &>/dev/null
```

Comment:
Where did this come from and why is it required? Why the for loop? 

Why would /etc/profile.d not have the bash_completion.sh?

Reply:
that is from the homebrew docs. that way you dont need the bash completion as stated in the next tab.